src/NServiceBus.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -79,7 +81,25 @@ public void Start(PushRuntimeSettings limitations)
             connection = connectionFactory.CreateConnection($"{settings.InputQueue} MessagePump");
 
             var channel = connection.CreateModel();
-            channel.BasicQos(0, (ushort)Math.Min(ushort.MaxValue, Convert.ToUInt16(limitations.MaxConcurrency) * prefetchCountPerMessageProcessor), false);
+
+            long prefetchCount;
```

Comment:
Went with a long here to make sure we don't overflow when doing the multiplier calculation. Sure, not likely to be an actual problem, but seemed easy enough to make sure.